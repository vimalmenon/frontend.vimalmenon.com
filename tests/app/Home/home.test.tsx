import Home from '../../../src/app/page';
import { render } from '@testing-library/react';

describe('Home', () => {
  it('Snapshot testing', async () => {
    // @ts-expect-error: Known error
    const { container } = render(await Home({ params: {} }));

    expect(container).toMatchSnapshot();
  });
});
