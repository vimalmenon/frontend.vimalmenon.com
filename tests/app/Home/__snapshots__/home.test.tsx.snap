// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot testing 1`] = `
<div>
  <div
    class="MuiBox-root css-j7qwjs"
  >
    <div
      class="MuiBox-root css-0"
    >
      This is Home page
    </div>
    <div
      class="MuiBox-root css-1bcth50"
    >
      <div
        class="MuiBox-root css-0"
      >
        API
      </div>
      <div
        class="MuiBox-root css-0"
      />
    </div>
    <div
      class="MuiBox-root css-1bcth50"
    >
      <div
        class="MuiBox-root css-0"
      >
        Env
      </div>
      <div
        class="MuiBox-root css-0"
      >
        Dev
      </div>
    </div>
  </div>
</div>
`;
